source/js/download.js

```diff
@@ -0,0 +1,62 @@
+$(document).ready(function($) {
+    'use strict';
+
+    showdown.setFlavor('github');
+    // Create markdown converter, starting all <h1> tags instead at <h3>
+    var converter = new showdown.Converter({headerLevelStart: 3}),
+        display_error_msg = function() {
+            var downloads = $('#downloads'),
+                more_details = $('#more-details');
+            
+            downloads.html('<h2>Sorry, we appear to be having difficulties fetching our binaries</h2>' + more_details[0].outerHTML);
+        };
+
+    $.ajax({
+        url:'/github.php',
+        success: function(data) {
+            var jdata = JSON.parse(data),
```

Comment:
There's still the issue here, what if the response cannot be parsed as JSON? I've tested and the page does show, but with no download links.